lib/modulesync/constants.rb

```diff
@@ -6,5 +6,6 @@ module Constants
     MODULESYNC_CONF_FILE = 'modulesync.yml'
     PROJ_ROOT            = './modules'
     HOOK_FILE            = '.git/hooks/pre-push'
+    GLOBAL_DEFAULTS_KEY  = :global
```

Comment:
This should probably be 'global' instead of :global so that users don't have to use a ruby symbol in the yaml file.